config-helper.js

```diff
@@ -50,6 +50,33 @@ module.exports = {
 			this.replaceMain({entry: config.entry[opts.chunk || 'main']}, replacement, opts);
 		}
 	},
+	replaceEntry: function (config, replacement, opts = {}) {
+		if (typeof replacement === 'string') {
```

Comment:
I don't think this type-checking necessary.

Reply:
When calling `replaceEntry` in CLI, parameter is `string` or `JSON object` so this statement needed.
https://github.com/enactjs/cli/pull/358/files#diff-3cd4247740b9464a85bfc18d3d7bd27b2b1102aa08106d0139c10a5afbda721bR268
(`opts.entry`: string, `app.entry`: JSON object)

Reply:
👌